### ISSUE TITLE
Make firewall rules for 'app' and 'mgmt' optional

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-google-gdm-templates#20]
Automatic creation of firewall rules for management and application access has been added in v2.0.0. The firewall rule creation cannot be controlled during the deployment of the GDM template.

This is problematic for existing deployment strategies as those might already take care of management and application access / firewall rule creation and now would need to delete the automatically created rules to prevent them to interfere with the existing strategies.

Hence my request would be to make both firewall rules optional, configurable by YAML paremeters.

For example using the same field for `application(Int)Port` but setting the value to `disabled`
```yaml
   applicationPort: 'disabled'
   applicationIntPort: 'disabled'
```
And introducing a new parameter for the management firewall creation.

**Comment by f5-gasingh** [comment]
Hi simonkowallik, I've added this request to our backlog and it is being track internally with ID -1293. Thank you!

**Comment by shyawnkarim** [comment]
Closing due to age. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-google-gdm-templates-v2).
